yarn.lock

```diff
@@ -4673,4 +4673,4 @@ zip-stream@^4.1.0:
   dependencies:
     archiver-utils "^3.0.4"
     compress-commons "^4.1.2"
-    readable-stream "^3.6.0"
+    readable-stream "^3.6.0"
```

Comment:
Why is this missing?

Maybe in your settings you don't include line break at the bottom.